python/paddle/fluid/tests/unittests/ir/inference/CMakeLists.txt

```diff
@@ -143,5 +143,6 @@ if (WITH_MKLDNN)
   set_tests_properties(test_mkldnn_conv_mish_fuse_pass PROPERTIES TIMEOUT 300)
   set_tests_properties(test_mkldnn_fc_mish_fuse_pass PROPERTIES TIMEOUT 300)
   set_tests_properties(test_mkldnn_fc_elementwise_add_fuse_pass PROPERTIES TIMEOUT 120)
+  set_tests_properties(test_mkldnn_conv_affine_channel_fuse_pass PROPERTIES TIMEOUT 60)
```

Comment:
I think this set_tests_properties is not necessary. If you remove this line, will all CIs pass too ?
Cause adding this line require Baidu's approval

Reply:
It is necessary. Without it, there is a timeout error in the Coverage CI.